compiler/rustc_passes/src/lang_items.rs

```diff
@@ -100,11 +100,12 @@ impl LanguageItemCollector<'tcx> {
     }
 
     fn collect_item(&mut self, item_index: usize, item_def_id: DefId) {
+        let lang_item = LangItem::from_u32(item_index as u32).unwrap();
```

Comment:
I started adding code to the bottom of this function before moving it to `collect_item_extended()`, and this is a leftover of that. I did not think it would hurt to have these two statements outside of the `if`, and so I left them there.

Reply:
Generally, it's better to declare variables only where they're needed. This helps for readability and reduces unnecessary computation.